src/analysis/retail/warlock/affliction/modules/spells/ShadowEmbrace.tsx

```diff
@@ -12,12 +12,12 @@ import Statistic from 'parser/ui/Statistic';
 import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
 import TalentSpellText from 'parser/ui/TalentSpellText';
 import UptimeBar from 'parser/ui/UptimeBar';
+import { shadowDefaultEmbraceModifier } from '../../constants';
+import { shadowDrainSoulEmbraceModifier } from '../../constants';
```

Comment:
You should import both constants in the same import.
```suggestion
import { shadowDefaultEmbraceModifier, shadowDrainSoulEmbraceModifier } from '../../constants';
```

Reply:
Done. My code doesn't look exactly like this but I think it's because prettier is formatting it differently.